### PR TITLE
Sql connections

### PR DESCRIPTION
**Pull request by Naawww**
Another simple module. Contains wrapped reference data library connections. These will not be accessible to those outside the organisation though, so is this a problem?

### CODE REVIEW

**Comment by murd0** [comment]
Hey @Naawww 

Can you please check the pipeline tasks that have failed and rectify them?

File: codonPython/tests/SQL_connections_test.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+'''test script for SQL_connections
+- test the connections can run a dummy script (SELECT 1 as [Code], 'test' as [Name])'''
+from codonPython import SQL_connections
```
Comment (murd0) [inline codonPython/tests/SQL_connections_test.py]: This import isn't needed

File: codonPython/tests/SQL_connections_test.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+import pandas as pd
+import pytest
+import codonPython.SQL_connections as conn
+
+@pytest.mark.parametrize("connection", 
+			 [conn.conn_DSS(),
+			  conn.conn_DSS2016UAT()
+			 ])
+def test_SELECT1(connection):
+	sql = """SELECT 1 as [Code], 'Test' as [Name]"""
```
Comment (murd0) [inline codonPython/tests/SQL_connections_test.py]: Variable not used

Comment (Naawww) [reply codonPython/tests/SQL_connections_test.py]: deleted "sql" variable.

File: codonPython/tests/SQL_connections_test.py (hunk 1 above)
Comment (Naawww) [reply codonPython/tests/SQL_connections_test.py]: removed "from codonPython import SQL_connections" as it was unused

**Comment by murd0** [comment]
To fix pyodbc install error for the pipeline: 
https://github.com/mkleehammer/pyodbc/issues/276
Add ` - sudo apt-get install unixodbc-dev` to `.travis.yml` under install

**Pull request merged**